plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8487,6 +8488,98 @@ public void testCreateAcidTableUnsupported()
         assertQueryFails("CREATE TABLE acid_unsupported WITH (transactional = true) AS SELECT 123 x", "FileHiveMetastore does not support ACID tables");
     }
 
+    @Test
+    public void testExtraProperties()
+    {
+        String tableName = "create_table_with_multiple_extra_properties_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE %s (c1 integer) WITH (extra_properties = MAP(ARRAY['extra.property.one', 'extra.property.two'], ARRAY['one', 'two']))".formatted(tableName));
+
+        assertQuery(
+                "SELECT \"extra.property.one\", \"extra.property.two\" FROM \"%s$properties\"".formatted(tableName),
+                "SELECT 'one', 'two'");
+
+        assertUpdate("CREATE TABLE %s_with_select (c1 integer) WITH (extra_properties = MAP(ARRAY['extra.property.one', 'extra.property.two'], ARRAY['one', 'two']))".formatted(tableName));
+
+        assertQuery(
+                "SELECT \"extra.property.one\", \"extra.property.two\" FROM \"%s_with_select$properties\"".formatted(tableName),
+                "SELECT 'one', 'two'");
+
+        assertUpdate("DROP TABLE %s".formatted(tableName));
+        assertUpdate("DROP TABLE %s_with_select".formatted(tableName));
+    }
+
+    @Test
+    public void testShowCreateWithExtraProperties()
```

Comment:
merge this with `testExtraProperties` so that you CREATE and DROP table fewer times